custom/icds_reports/reports/cas_reach_data.py

```diff
@@ -101,7 +110,7 @@ def get_data_for_daily_usage(date, filters):
                     'help_text': awcs_launched_help_text(),
                     'color': None,
                     'percent': None,
-                    'value': get_value(daily_yesterday if current_month_selected else awc_this_month_data, 'awcs'),
+                    'value': get_value(awc_this_month_data, 'awcs'),
```

Comment:
@dhiraj-beehyv feature flag check will also be applied to this line same as what you did on 86 line.